tests/js/spec/views/projectsDashboard/index.spec.jsx

```diff
@@ -47,150 +53,241 @@ describe('ProjectsDashboard', function () {
   });
 
   describe('empty state', function () {
-    it('renders with no projects', function () {
-      const noProjectTeams = [TestStubs.Team({projects: []})];
+    it('renders with no projects', async function () {
+      const noProjectTeams = [TestStubs.Team({isMember: false, projects: []})];
 
-      const wrapper = mountWithTheme(
+      render(
         <Dashboard
           teams={noProjectTeams}
           organization={org}
           params={{orgId: org.slug}}
         />,
-        routerContext
+        {routerContext}
```

Comment:
this should be called context, we might be able to remove it.

Try removing it first, otherwise it should be this
```suggestion
        {context: routerContext}
```